### ISSUE TITLE
черные линии на графике!

### ISSUE THREAD

**Issue by koroniova** [SBProX/SBProX-Dev#1145]
это скрин сбпроХ сверху и сбпро2 снизу, масштаб у них одинаковый, затемнения выключены в двух прогах, но в сбпроХ появляются какие то "прострельные" черные линии (красным указал)! т.е. в сбпро2 там видны кластера серые, а в сбпроХ как глюк черные линии!! на белом фоне это явно видно
![линии бел](https://user-images.githubusercontent.com/34831430/82535284-efb73a00-9b46-11ea-8db7-f0b554ef5ae2.jpg)
![линии](https://user-images.githubusercontent.com/34831430/82535294-f34ac100-9b46-11ea-8e58-55f2f4e0d120.jpg)

**Comment by affectus1** [comment]
Я уже писал сюда на эту тему мне сказали что все нормально и прикрыли тему

**Comment by SBProX** [comment]
Это нюансы движка отрисовки при низком масштабе.
Этот вовсе не линии, а неравномерно нарисованные объекты.
Когда будем переписывать отрисовку постараемся учесть эти нюансы, НО во многом зависит от разрешения и масштабировать экрана, из-за чего округляются координаты объектов, что и вызывает такой эффект.

**Comment by koroniova** [comment]
> Это нюансы движка отрисовки при низком масштабе.
> Этот вовсе не линии, а неравномерно нарисованные объекты.
> Когда будем переписывать отрисовку постараемся учесть эти нюансы, НО во многом зависит от разрешения и масштабировать экрана, из-за чего округляются координаты объектов, что и вызывает такой эффект.

Да, эти линии появляются при определенном масштабе, но это не нормально...... Приходится или еще мельче делать или наоборот крупнее. 
Спасибо за то что разберетесь в дальнейшем
